client/src/components/charts/DashMap.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
+import * as chart from '../../utilities/chart';
+
+require('../../../node_modules/leaflet/dist/leaflet.css');
+require('../../styles/DashMap.scss');
+
+
+export default function DashMap({ visualisation, datasets }) {
+  const chartData = chart.getChartData(visualisation, datasets);
+
+  return (
+    <div className="DashMap dashChart">
+      <Map
+        center={[0, 0]}
+        zoom={2}
+        scrollWheelZoom={false}
```

Comment:
Do we want to enable this feature? I think it easier to zoom on a particular area if this is enabled? An example of other maps that use scroll to zoom:

> When the user scrolls with the mouse over the map, map zooms in and out (as it currently does).

https://github.com/akvo/akvo-flow/issues/1475

I would propose: `scrollWheelZoom='center'`